### PR TITLE
DEV026 Create index and show pages for place model

### PR DESCRIPTION
**Pull request by als6k**
Create index and show pages for place model
[card](https://trello.com/c/88mPnRll/70-dev026-create-index-and-show-pages-for-place-model)

### CODE REVIEW

File: app/assets/stylesheets/active_admin.scss [hunk 1]
```diff
@@ -1,2 +0,0 @@
-@import "active_admin/mixins";
```
Comment (Maxxx911) [inline app/assets/stylesheets/active_admin.scss]: Don't delete this lines

File: app/javascript/components/places/PlaceIndex.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import withStyles from "@material-ui/core/styles/withStyles";
+import landingPageStyle from "../shared/style/landingPage";
+import Parallax from "../shared/parallax.jsx";
+import Grid from '@material-ui/core/Grid';
+import PlacesCard from "../shared/card_places";
+
+class PlaceIndex extends React.Component {
+  constructor(props) {
```
Comment (Maxxx911) [inline app/javascript/components/places/PlaceIndex.jsx]: Are you sure that we need constructor in here? constructor usage is out dated

File: app/javascript/components/places/PlaceShow.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+import Grid from '@material-ui/core/Grid';
+import MyImage from "../shared/image";
+import Title from "../shared/title";
+import SmallText from "../shared/small_text";
+import Description from "../shared/description";
+import EventsCard from "../shared/card_events";
+import ImageCarousel from "../shared/image_carousel";
+
+class PlaceShow extends React.Component {
+  constructor(props) {
```
Comment (Maxxx911) [inline app/javascript/components/places/PlaceShow.jsx]: Are you sure that we need constructor in here? constructor usage is out dated

File: app/javascript/components/places/PlaceShow.jsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  render() {
+    const {classes, place} = this.props;
+    return (
+      <div>
+        <div className={classNames(classes.main, classes.mainRaised)}>
+          <div className={classes.container}>
+            <React.Fragment>
+              <Grid container>
+                <Grid item xs={12} sm={12} md={7} lg={7} xl={7}>
+                  <MyImage width={"90%"} height={"90%"} src={"https://upload.wikimedia.org/wikipedia/ru/e/ef/%D0%A1%D0%BA%D1%80%D0%B8%D0%BD%D1%88%D0%BE%D1%82_%D1%81%D0%B5%D1%80%D0%B2%D0%B8%D1%81%D0%B0_%D0%AF%D0%BD%D0%B4%D0%B5%D0%BA%D1%81.%D0%9A%D0%B0%D1%80%D1%82%D1%8B.png"}></MyImage>
```
Comment (andreydro) [inline app/javascript/components/places/PlaceShow.jsx]: let's use image from server

Comment (als6k) [reply app/javascript/components/places/PlaceShow.jsx]: там будет карта, когда мы ее прикрутим и эта картинка там сейчас просто для наглядности

**Pull request merged**